catalog/opensearch/catalog-opensearch-api/src/main/java/org/codice/ddf/opensearch/source/PointRadiusSearch.java

```diff
@@ -38,4 +40,25 @@ public double getLat() {
   public double getRadius() {
     return radius;
   }
+
+  @Override
+  public boolean equals(Object object) {
+    if (this == object) {
+      return true;
+    }
+
+    if (!(object instanceof PointRadiusSearch)) {
```

Comment:
❓ Should we check for null input before doing any other comparisons or does this take care of null here?

Reply:
`instanceOf` already checks for `null`.